business/apps_test.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/kiali/kiali/config"
 	"github.com/kiali/kiali/kubernetes/kubetest"
 	"github.com/kiali/kiali/prometheus/prometheustest"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
trivial - but this should be formatted as per standard (put this in the middle set of imports - i.e after line 13).